package.json

```diff
@@ -74,14 +75,14 @@
     "@types/marked": "^5.0.0",
     "@types/natural-compare": "^1.4.1",
     "@types/ncp": "^2.0.5",
-    "@types/node": "^20.0.0",
+    "@types/node": "18.16.9",
     "@types/semver": "^7.5.0",
     "@types/tmp": "^0.2.3",
     "console-fail-test": "^0.2.3",
     "cross-fetch": "^4.0.0",
     "cspell": "^7.0.0",
     "downlevel-dts": ">=0.11.0",
-    "eslint": "^8.47.0",
+    "eslint": "8.48.0",
```

Comment:
```suggestion
    "eslint": "^8.48.0",
```

Integration tests're faililin because of pinned `eslint` version:
- Stylistic rules were deprecated in `v8.53.0` ([blog post](https://eslint.org/blog/2023/11/eslint-v8.53.0-released/))
- ESLint version for integration tests is taken from the root package.json https://github.com/typescript-eslint/typescript-eslint/blob/3d729303dd5cfd18ccb823e96f0e8761fda6adab/packages/integration-tests/tools/integration-test-base.ts#L27
- Since it is pinned, yarn resolves ESLint as `8.48.0` rather than `>=8.53.0`
- Tests fail because stylistic rules aren't marked as deprecated
```
FAIL  tests/vue-sfc.test.ts (21.592 s)
  ● vue-sfc › should lint successfully
    expect(received).toMatchSnapshot()
    Snapshot name: `vue-sfc should lint successfully 1`
    - Snapshot  - 18
    + Received  +  3
    @@ -53,[16](https://github.com/typescript-eslint/typescript-eslint/actions/runs/7426374087/job/20210070340?pr=8194#step:5:17) +53,11 @@
        },
      });
      </script>
      ",
          "suppressedMessages": [],
    -     "usedDeprecatedRules": [
    -       {
    -         "replacedBy": [],
    -         "ruleId": "semi-spacing",
    -       },
    -     ],
    +     "usedDeprecatedRules": [],
          "warningCount": 0,
        },
        {
          "errorCount": 1,
          "fatalErrorCount": 1,
    @@ -93,16 +88,11 @@
        }
      }
```

Reply:
Thanks @auvred!